internal/metrics/info_test.go

```diff
@@ -65,11 +65,11 @@ func Test_AddPolicyCountCallback(t *testing.T) {
 func Test_SetConfigChecksum(t *testing.T) {
 	registry = newMetricRegistry()
 
-	wantValue := int64(42)
+	wantValue := float64(42)
```

Comment:
I can't believe I missed the answer to the ultimate question in life on the first review 😆

Reply:
Most important value to test.